python/plugins/processing/tests/AlgorithmsTestBase.py

```diff
@@ -142,6 +142,14 @@ def load_param(self, param):
                 return [self.load_param(p) for p in param['params']]
             elif param['type'] == 'file':
                 return self.filepath_from_param(param)
+            elif param['type'] == 'interpolation':
+                prefix = processingTestDataPath()
+                tmp = ''
```

Comment:
Could this use the `ParameterInterpolationData` class for this?

Something like

    parameter = ParameterInterpolationData()
    parameter.setValue(param['name'])
    parameter.setPrefixPath(processingTestDataPath())